scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -785,6 +795,9 @@ private void processAlertRecord(final Record record, final Connection connection
 					errorHandler.onError(alert.getPeer(), alert.getLevel(), alert.getDescription());
```

Comment:
This sounds acceptable. I tried a longtime ago to use ErrorHandler and this is not so easy as you don't have so much context. Handshake errors available on `MessageCallback.onError` is really better !